src/constants/actionTypes.js

```diff
@@ -95,7 +95,7 @@ export const LAYER_EDIT_RELATIONSHIP_LINE_COLOR_SET =
 export const LAYER_EDIT_ORGANISATION_UNITS_SET =
     'LAYER_EDIT_ORGANISATION_UNITS_SET'
 export const LAYER_EDIT_ORGANISATION_UNIT_GROUP_SET =
-    'LAYER_EDIT_ORGANISATION_UNIT_GROUP_SET'
+    'LAYER_EDIT_ORGANISATION_UNIT_GROUP_SET' // Facility TODO: Distinguish frome above
```

Comment:
@turban Should this TODO be a jira issue? If not, could you give me some more detail?

Reply:
Please remove this comment.